cmd/init.go

```diff
@@ -70,10 +70,17 @@ validated and configured.`,
 			}
 		}
 
-		if os.Getenv("GITHUB_AUTH_TOKEN") != "" {
-			viper.Set("github.token", os.Getenv("GITHUB_AUTH_TOKEN"))
-		} else {
-			log.Fatal("cannot create a local cluster without a github auth token. please export your GITHUB_AUTH_TOKEN in your terminal.")
+		providerValue, err := cmd.Flags().GetString("git-provider")
+		if err != nil {
+			return err
+		}
+
+		if providerValue == "github" && cloudValue == flagset.CloudAws {
```

Comment:
```suggestion
		if providerValue == "github" && cloudValue == flagset.CloudK3d {
```

Reply:
id this gate for the cloud or local story?

Reply:
@jarededwards is for the cloud story, local story won't use use the github token through environment variable, it will generate a token using oauth app